models/hostgroupDataSource.go

```diff
@@ -0,0 +1,29 @@
+package models
+
+import "github.com/hashicorp/terraform-plugin-framework/types"
+
+// HostGroupDataSource - HostGroupDataSource properties
+type HostGroupDataSource struct {
+	ID                        types.String                `tfsdk:"id"`
+	Name                      types.String                `tfsdk:"name"`
+	Description               types.String                `tfsdk:"description"`
+	HostConnectivity          types.String                `tfsdk:"host_connectivity"`
+	HostConnectivityL10n      types.String                `tfsdk:"host_connectivity_l10n"`
+	Hosts                     []Hosts                     `tfsdk:"hosts"`
+	MappedHostGroups          []MappedHostGroup           `tfsdk:"mapped_host_groups"`
+	HostVirtualVolumeMappings []HostVirtualVolumeMappings `tfsdk:"host_virtual_volume_mappings"`
```

Comment:
Dont see the struct for HostVirtualVolumeMappings

Reply:
HostVirtualVolumeMappings struct is part of host hostDataSource.go